apps/storybook/src/overlays/AlertDialog.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { AlertDialog, Button, Flex } from "@optiaxiom/react";
+import { expect, screen, userEvent, within } from "@storybook/test";
+import { useState } from "react";
+
+const meta: Meta<typeof AlertDialog> = {
+  argTypes: {
+    action: { control: "text" },
+    cancel: { control: "text" },
+    onAction: { action: "onAction" },
+    onCancel: { action: "onCancel" },
+    title: { control: "text" },
+  },
+  component: AlertDialog,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof AlertDialog>;
+
+const Template = (args: React.ComponentProps<typeof AlertDialog>) => {
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  return (
+    <Flex>
+      <Button onClick={handleOpen}>Open Dialog</Button>
+      <AlertDialog
+        {...args}
+        onAction={() => {
+          args.onAction();
+          handleClose();
+        }}
+        onCancel={() => {
+          args.onCancel();
+          handleClose();
+        }}
+        open={open}
+      >
+        {args.children}
+      </AlertDialog>
+    </Flex>
+  );
+};
+
+export const Default: Story = {
+  args: {
+    action: "Confirm",
+    cancel: "Cancel",
```

Comment:
let's not provide defaults here

just as a general rule story args should be very deliberate - testing specific stuff

we can have a separate story that changes the action label - and then have a `play` method that specifically asserts that